### DISCUSSION TITLE
Using `<Label />` and `<Select />` together?

### DISCUSSION THREAD

**Discussion by bnussman** [tamagui/tamagui#2153] [General]
Hello 👋🏼 

I want to create an accessible `<Select />` component with a `<Label />`. Whenever I try the following:

```tsx
<Label htmlFor="make">
  Make
</Label>
<Select id="make">
...
</Select>
```

 It does not appear to work. Clicking the label does not focus/activate the select. 
 
I'm hoping to get this to work in Expo Web.  Could someone provide me an example of how to make this work? If this isn't easily possible, should I create an issue?

**Comment by likeSo** [comment]
I have the same problem🙋

**Comment by natew** [comment]
Looking at this one today

**Reply by natew** [reply]
Should be fixed in 36c816f9a1e96d0cb8e0f2df2652abc8ca25930c and b515541e3387afd3caacc15757dc6d3bb89a3a28

**Reply by bnussman** [reply]
Looks much better, but I'm not sure the `Native` web select works as expected still.
